core/templates/dev/head/app.js

```diff
@@ -1220,3 +1220,20 @@ oppia.factory('codeNormalizationService', [function() {
     }
   };
 }]);
+
+// Service for retrieving algorithm id of classifier for a given interaction id.
+oppia.factory('algorithmIdRetrievalService', [function() {
+  // Maintain the Interaction-Classifier mapping as defined in feconf.
+  var interactionClassifierMapping = {
```

Comment:
@pranavsid98 would it be better to pass algorithmId with classifier data itself? That way we can use appropriate algorithm using which classifier data is obtained. It also gives us backward compatibility and we can use old approach if we have old classifier data. @seanlip what do you think?